suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -0,0 +1,363 @@
+                - service_type: crash
+                  placement:
+                    host_pattern: "*"
+                  extra_container_args:
+                    - "--cpus=1"
+                - service_type: osd
+                  service_id: all-available-devices
+                  encrypted: "true"                     # boolean as string
+                  placement:
+                    host_pattern: "*"
+                  data_devices:
+                    all: "true"                         # boolean as string
+                  extra_container_args:
+                    - "--cpus=2"
+          - config:
+              command: shell
+              args:                 # sleep to get all services deployed
+                - sleep
+                - "300"
+  - test:
+      name: MDS Service deployment with spec
+      desc: Add MDS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574728
+      config:
+        steps:
+          - config:
+              command: shell
+              args:               # arguments to ceph orch
+                - ceph
+                - fs
+                - volume
+                - create
+                - cephfs
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: mds
+                  service_id: cephfs
+                  placement:
+                    label: mds
+                  extra_container_args:
+                    - "--cpus=1"
+          - config:
+              command: shell
+              args:              # sleep to get all services deployed
+                - sleep
+                - "120"
+  - test:
+      name: NFS Service deployment with spec
+      desc: Add NFS services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574729
+      config:
+        steps:
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: nfs
+                  service_id: nfs-rgw-service
+                  placement:
+                    nodes:
+                      - node4
+                  spec:
+                    port: 2049
+                  extra_container_args:
+                    - "--cpus=1"
+          - config:
+              command: shell
+              args:              # sleep to get all services deployed
+                - sleep
+                - "120"
+  - test:
+      name: Setup destination node for SNMP traps
+      desc: Install snmptrapd tool and install CEPH MIB on destination node
+      module: snmp_destination.py
+      polarion-id: CEPH-83574775
+      config:
+        node: node3
+        specs:
+              - service_type: snmp-destination
+                spec:
+                  credentials:
+                    snmp_v3_auth_username: myadmin
+                    snmp_v3_auth_password: mypassword
+  - test:
```

Comment:
are CPU extra_container_args supported for SNMP?

Reply:
can we include, if it is supported.

Reply:
Yes I have included it. It is not throwing any error while including it as extra container args but not sure if it actually is getting considered. I'll be testing it out once again.